xbmc/GUIInfoManager.cpp

```diff
@@ -7182,7 +7187,7 @@ bool CGUIInfoManager::GetBool(int condition1, int contextWindow, const CGUIListI
     case PLAYER_PLAYING:
       {
         float speed = g_application.m_pPlayer->GetPlaySpeed();
-        bReturn = (speed >= 0.75 && speed <= 1.55);
+        bReturn = (speed == 1.0);
```

Comment:
if it does not generate a warning, I see no benefit in adding this silly f. I'll leave this to whoever wants to be the maintainer of this part of the app that would need a rewrite from scratch anyway